src/main/java/org/mockito/plugins/InlineMockMaker.java

```diff
@@ -37,8 +37,8 @@ public interface InlineMockMaker extends MockMaker {
     void clearMock(Object mock);
 
     /**
-     * Cleans up internal state for all existing mocks. You may assume there won't be any interaction to mocks created
-     * previously after this is called.
+     * Cleans up internal state for all existing mocks. Attempts to interact with mocks after this
+     * is called will throw {@link org.mockito.internal.framework.DisabledMockException}
```

Comment:
The JavaDoc task fails on referencing this type. That's because we exclude `org.mockito.internal` classes in our JavaDoc. That makes me think: we should expose this exception in our public API instead. I think we should put it in `src/main/java/org/mockito/exceptions/misusing`

Reply:
Sounds good, did so!